ariadne/cardano/src/Ariadne/Wallet/Face.hs

```diff
@@ -86,6 +86,7 @@ data WalletFace =
         Bool -> WalletReference -> [LocalAccountReference] ->
         InputSelectionPolicy -> NonEmpty TxOut -> IO TxId
     , walletBalance :: IO Coin
+    , walletCalcTotal :: [Coin] -> IO Coin
```

Comment:
The command is called `sum-coins`, why not call the function the same way? :thinking: